integrations/symfony-bundle.rst

```diff
@@ -0,0 +1,176 @@
+Symfony Bundle
+==============
+
+The usage documentation is split into two parts. First we explain how to configure the bundle in an application. The second part is for developing reusable Symfony bundles that depend on an HTTP client defined by the Httplug interface.
```

Comment:
the second part is just a very short section for now. but i think we should nonetheless mention it like this.

but before the organizational notes, we should have a first paragraph saying something like

"The HttplugBundle allows to configure clients with specific settings and plugin chains. It provides services for the tools found in the php-http universe."